contracts/BitGuildAccessAdmin.sol

```diff
@@ -0,0 +1,147 @@
+pragma solidity ^0.4.24;
+
+
+/**
+ * @title BitGuildAccessAdmin
+ * @dev Allow two roles: 'owner' or 'operator'
+ *      - owner: admin/superuser (e.g. with financial rights)
+ *      - operator: can update configurations
+ */
+contract BitGuildAccessAdmin {
+    address public owner;
+    address[] public operators;
+
+    uint public MAX_OPS = 10; // Default maximum number of operators allowed
+
+    mapping(address => bool) public isOperator;
+
+    event OwnershipTransferred(
+        address indexed previousOwner,
+        address indexed newOwner
+    );
+    event OperatorAdded(address operator);
+    event OperatorRemoved(address operator);
+
+    /// @dev The BitGuildAccessAdmin constructor: sets owner to the sender account
```

Comment:
Nit: why three /? Why not 2? Here and everywhere

Reply:
Mixed different conventions. Changed it back to 2 now.